### PR TITLE
Add release notes for v0.4

### PR DESCRIPTION
**Pull request by nott**

### CODE REVIEW

**Comment by nott** [comment]
I expect the tests to be red due to an issue with crates.io certificate

File: src/main.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@ use std::fs::File;
 
 fn main() {
     let global_matches = App::new("Cobalt")
-        .version("0.3.0")
+        .version(crate_version!())
```
Comment (johannhof) [inline src/main.rs]: nice!

**Comment by nott** [comment]
@johannhof do you mind if i use crate_version macro so we have a single version number in the source code?

**Comment by johannhof** [comment]
Not at all.

**Pull request merged**